cves/kernel/CVE-2020-25669.yml

```diff
@@ -192,8 +206,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: In the reports, no violation of specification was specified; This bug was due to carelss 
```

Comment:
Typo of "careless". Also these sentences are in general a little wonky. Just missing the word "a/any" I think. So, "no violation of any specification" and "due to a careless mistake"

Reply:
Fixed!